framework/include/controls/ControlInterface.h

```diff
@@ -54,29 +38,25 @@ class ControlInterface
 {
 protected:
 
+  ///@{
   /**
-   * Obtain a writeable reference to a single parameter given a InputParameter name
+   * Obtain a ControllableParameter class for accessing parameter values given input file syntax.
    */
   template<typename T>
-  T & getControlParam(const std::string & name);
-
-  /**
-   * Obtain a writeable reference to a single parameter given a specific name
-   */
+  ControllableParameter<T> getControlParam(const std::string & name);
```

Comment:
If it's going to return `ContrallABLE` then the function should probably be called `getControllableParam()`